packages/controllers/src/controllers/ApiController.ts

```diff
@@ -19,11 +19,17 @@ import { ConnectorController } from './ConnectorController.js'
 import { EventsController } from './EventsController.js'
 import { OptionsController } from './OptionsController.js'
 
+const CORE_UNSUPPORTED_WALLET_IDS = [
+  '1ca0bdd4747578705b1939af023d120677c64fe6ca76add81fda36e350605e79',
```

Comment:
How do we maintain this? I mean where do we get this array of wallets from? For the future we should add more if necessary?

Reply:
This should be done proper API-side, this list should not be expanded.